workloads/web.hbs.md

```diff
@@ -57,3 +57,38 @@ tanzu apps workload apply tanzu-java-web-app --type=web
 
 You can use this to test which applications can function well as serverless web applications,
 and which are more suited to the `server` application style.
+
+## <a id="using"></a> Communication between `web` workloads
```

Comment:
You will need a unique anchor id. The id "using" is already taken by other title